src/test/resources/templates/htmxRequest.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>htmx request</title>
+</head>
+<body>
+<!--/*@thymesVar id="#htmxRequest" type="HtmxRequest"*/-->
+<div id="webRequest" th:if="${#request.getHeader('HX-Request')}">It is an htmx Request using the underlying request</div>
```

Comment:
I had thought the sub-classing was needed to avoid removing the Spring standard ones, but this test shows that `#request` is still available.